Add Ares and Phobos Version Numbers

CnCRAZER opened the pull request:
- Adds version numbers for Ares and Phobos.
- Changes URL's to documentation

<img width="1147" height="812" alt="image" src="https://github.com/user-attachments/assets/190748aa-1965-4873-aaac-e09d0c0de751" />

<img width="1227" height="849" alt="image" src="https://github.com/user-attachments/assets/abe47901-01f5-4906-82e6-16f60c6f5bd3" />

CnCRAZER committed:
Add Powered By Version Numbers

CCCP84 commented:
What is RVGE? RAVAGE? He has some kind of obsession with adding his name to everything..

CnCRAZER commented:
> What is RVGE? RAVAGE? He has some kind of obsession with adding his name to everything..

Yeah they're the themes he made lol

CCCP84 commented:
Are these version numbers even necessary? Who would need this information? Probably just the client version number would be enough, and that link would provide information about the Ares and Phobos versions.
I don't think there's any need to overload the menu screen with unnecessary information.

CCCP84 commented:
Do we need a check for updates button? As far as I understand, the client automatically checks for updates every time it's launched. If necessary, you can do this in the menu.

CnCRAZER commented:
The first time anyone wants to add anything from one of the engine extensions they're gonna go to the docs and see a bunch of different versions. They'll either try adding stuff not supported or come to discord and ask. I don't believe the info provided in the way my screenshots do is overwhelming tbh

CCCP84 commented:
I think in 95 percent of cases, it's the players who log into the client, not the modders, and in turn, the modders. If they've figured out how to make mods, they can follow the link by clicking on the client version number, which will provide information about the version of Ares and Phobos.

CCCP84 commented:
I believe everything should be concise. There's no need to overload the client with unnecessary details, especially the main menu.

Metadorius commented:
> If they've figured out how to make mods, they can follow the link by clicking on the client version number

I think this is unnecessary gatekeeping. Being presented with specific version number is good. And being a modder doesn't mean you always pay attention to such things. I experienced this a lot because people keep using latest version docs and wondering why new stuff doesn't work. No need to make the situation even worse.

Metadorius commented on package/Resources/MainMenu.ini:
```diff
@@ -168,6 +161,20 @@ Size=600,30
+Location=465,577
+Size=600,30
+FontIndex=1	;4
+RemapColor=247,228,99
```
Can you make it anchored to the right? So that when we change to 0.4 we won't need to adjust this string location. And maybe let's link to the docs of the specific version (not sure if changelog is better, just the doc version should do?).

CnCRAZER replied:
Yeah I'll take care of this in the next day or so

CCCP84 commented:
The menu should be primarily aimed at the average user. Overloading it with information about the versions of all included projects is more technical and specialized information. I don't think it looks good, but that's just my opinion.

Alternatively, version information can be displayed by hovering over the Phobos or Ares emblem.
Or on the page that will appear when you click on these links

CnCRAZER committed:
Revert "Add Powered By Version Numbers"

This reverts commit 78dc18e3ed339a887b8198bc09f9f38bf3c96414.

CnCRAZER committed:
Update MainMenu.ini

CnCRAZER committed:
Merge branch 'CnCNet:develop' into Version-Additions

CnCRAZER commented:
> The menu should be primarily aimed at the average user. Overloading it with information about the versions of all included projects is more technical and specialized information. I don't think it looks good, but that's just my opinion.
> 
> 
> 
> Alternatively, version information can be displayed by hovering over the Phobos or Ares emblem.
> 
> Or on the page that will appear when you click on these links

I reverted the original proposal and put the version in the tooltip of the emblems and changed the URL's to point to their appropriate documentations

Thanks for the feedback

CCCP84 commented:
Yes, this also looks terrible.
Maybe you could just write "Phobos 0.4" there?
or don't write anything at all, but have this version on the page the link leads to

CCCP84 commented:
And why do we need this button to check for updates?

CCCP84 commented:
Perhaps it would be possible to make the Phobos emblem and so on take on colors when the cursor hovers over them?

CnCRAZER commented:
> Yes, this also looks terrible.
> 
> Maybe you could just write "Phobos 0.4" there?
> 
> or don't write anything at all, but have this version on the page the link leads to

The current technical version is 0.4-RC2, with RC3 pending iirc.

CCCP84 commented:
> The current technical version is 0.4-RC2, with RC3 pending iirc.

So, at least write there just "Phobos 0.4-RC2" without this long text, which is unnessesary and spoils the view.
Well, or as I already said, in my opinion the best option would be to indicate the version on the page and here do without the inscription

CnCRAZER committed:
Update Phobos Version and docs url

CnCRAZER commented:
Updated phobos version description and url to point to `0.4`

Belonit commented:
I would prefer to see description on the first line and version on the second

CCCP84 commented:
Is it possible to replace the image of the emblem when hovering over it?

<img width="829" height="626" alt="menu" src="https://github.com/user-attachments/assets/308325ea-cd2e-442a-8ea6-6a7856d943ce" />

MahBoiDeveloper commented:
> Is it possible to replace the image of the emblem when hovering over it?

Do you want to edit logo for every client update? Why we cannot just use tooltips for it when cursor hovering Ares/Phobos logo?

MahBoiDeveloper commented:
@CnCRAZER what do you think if to the tooltip would be added text "Click to the logo to follow the release page" or something?

CCCP84 commented:
> Do you want to edit logo for every client update? Why we cannot just use tooltips for it when cursor hovering Ares/Phobos logo?

maybe add a separate line for this?

MahBoiDeveloper commented:
> maybe add a separate line for this?

Version info of the client parts is not so important to see every time when you launch the client.

CCCP84 commented:
> Version info of the client parts is not so important to see every time when you launch the client.

Exactly. I'm of the same opinion. But if it still needs to be done, it's possible to make it so that this line is displayed only when the cursor hovers over the corresponding emblem.

CCCP84 commented:
This is how it should be:

https://github.com/user-attachments/assets/d9c7ca99-f559-4911-855b-a028764cd219

Version number - this is not a picture, it's a string

CCCP84 commented:
If you like it, tell me I'll do PR

MahBoiDeveloper commented:
@CnCRAZER it is the bad design to merge version string into logo.

CCCP84 commented:
> @CnCRAZER it is the bad design to merge version string into logo.

You can ask me about design. I have high taste and a subtle sence of aesthetics.
*This line is not integrated into the logo.

CnCRAZER commented:
> This is how it should be:
> 
> 
> 
> https://github.com/user-attachments/assets/d9c7ca99-f559-4911-855b-a028764cd219
> 
> 
> 
> Version number - this is not a picture, it's a string
> 
> 

Yeah go ahead and pr something like this

CCCP84 commented:
I hope you weren't offended.. 
But these long texts in tips really looks awkward.
It would be nice if someone else also wrote their opinion.

CnCRAZER commented:
No offense taken at all. That video you made looked really good

CCCP84 commented:
Do you know the Ares logo? Is it in colors or is it black and white? Like Phobos, it would be nice to give it color when you hover over it.
Well, either we'll have to come up with some colors for it. maybe something neutral and not flashy.
The cncnet logo can be given a metallic tint.
